lib/question.js

```diff
@@ -1534,6 +1534,11 @@ module.exports = {
       locals.showTrueAnswer = true;
     }
 
+    // Manually disable correct answer panel
+    if (question && question.hide_correct_answer) {
```

Comment:
Nit (though I think `question` should always be defined here, so checking if it's falsy if probably unnecessary?)
```suggestion
    if (question?.hide_correct_answer) {
```